packages/dds/matrix/src/handlecache.ts

```diff
@@ -72,11 +72,10 @@ export class HandleCache implements IVectorConsumer<Handle> {
 	/**
 	 * Used by {@link HandleCache.cacheMiss} to retrieve handles for a range of positions.
 	 */
-	private getHandles(start: number, end: number): Handle[] {
+	private getHandles(start: number, end: number, handles: Handle[]): Handle[] {
 		// TODO: This can be accelerated substantially using 'walkSegments()'.  The only catch
 		//       is that
 
-		const handles: Handle[] = [];
 		const { vector } = this;
 
 		for (let pos = start; pos < end; pos++) {
```

Comment:
I'd think we want to remove `return handles;` statement at the bottom, as not used / useful?